### ISSUE TITLE
Installation fails on Ubuntu impish

### ISSUE THREAD

**Issue by peterdemaeyer** [Azure/azure-functions-core-tools#2860]
Installation using Ubuntu 21.10 impish fails, using any of the installation methods, also when using 21.04 as fallback.

1. Using the procedure in the `README.md`: `https://packages.microsoft.com/config/ubuntu/21.10/packages-microsoft-prod.deb` fails with `404: Not Found`. The obvious first problem is that the directory `https://packages.microsoft.com/config/ubuntu/21.10/` exists, but there is no `.deb` file.
2. Fallback to 21.04 gets me further but still fails to locate the package:
```
wget -q https://packages.microsoft.com/config/ubuntu/20.04/packages-microsoft-prod.deb
sudo dpkg -i packages-microsoft-prod.deb
sudo apt-get update
sudo apt-get install azure-functions-core-tools-4
...
E: Unable to locate package azure-functions-core-tools-4
```
So the package is missing.
3. Using the procedure in the [Microsoft docs](https://docs.microsoft.com/en-us/azure/azure-functions/functions-run-local?tabs=v4%2Clinux%2Ccsharp%2Cportal%2Cbash%2Ckeda):
```
curl https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > microsoft.gpg
sudo mv microsoft.gpg /etc/apt/trusted.gpg.d/microsoft.gpg
sudo sh -c 'echo "deb [arch=amd64] https://packages.microsoft.com/repos/microsoft-ubuntu-$(lsb_release -cs)-prod $(lsb_release -cs) main" > /etc/apt/sources.list.d/dotnetdev.list'
sudo apt-get update
sudo apt-get install azure-functions-core-tools-4
...
Err:11 https://packages.microsoft.com/repos/microsoft-ubuntu-impish-prod impish Release
  404  Not Found [IP: 13.80.99.124 443]
Reading package lists... Done
E: The repository 'https://packages.microsoft.com/repos/microsoft-ubuntu-impish-prod impish Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package azure-functions-core-tools-4
```

The update is the first to fail, so the install fails as well.

I'm out of ideas, I'm stuck, unable to start experimenting with Azure Functions.

**Comment by peterdemaeyer** [comment]
I had to fall back to as far as focal, and then installation succeeded using the procedure from the Microsoft docs.

**Comment by michaelpeng36** [comment]
@peterdemaeyer thank you for bringing up this issue. I have just updated the relevant repository. Could you try installing again using the instructions from the `README.md` and Ubuntu 21.10 impish? Let me know if this issue persists.

**Comment by peterdemaeyer** [comment]
Using the instructions in the `README.md` it still fails, but using the procedure in the Microsoft docs it works now for impish.

**Comment by DenzoNL** [comment]
I also ran into this issue today with Ubuntu 22.04. Falling back to the Focal version also worked for me, in case anyone else has this issue. Official support would be nice though :) If there's anything I can do to help, let me know

**Comment by michaelpeng36** [comment]
@peterdemaeyer It looks like the `.deb` file was added to the `https://packages.microsoft.com/config/ubuntu/21.10/` directory. Would you be able to verify that the `README.md` instructions now work for Impish?

@DenzoNL, support for Jammy seems to have been resolved recently, as indicated on #3037. Please let me know if this is not the case for you!
